homework-submission/week3/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import './App.css';
+import Description from './ToDo/Description';
+import DeadLines from './ToDo/DeadLine';
+import DoneCheckBox from './ToDo/DoneCheckBox';
+import todos from './todoList';
+/** WEEK 3 */
+import NewToDoComponent from './ToDo/NewToDoComponent';
+import ControlsComponent from './ToDo/ControlsComponent';
+
+let oldDescription = '';
```

Comment:
why do you create this variable
just remove it from everywhere in this file, and your code will work fine